docs/tls/kubernetes/README.md

```diff
@@ -0,0 +1,80 @@
+# How to secure access to Minio on Kubernetes with TLS [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+This document explains how to configure Minio server with TLS certificates on Kubernetes.
+
+## 1. Prerequisites
+
+- Familiarity with [Minio deployment process on Kubernetes](https://docs.minio.io/docs/deploy-minio-on-kubernetes).
+
+- Kubernetes cluster with `kubectl` configured.
+
+- Acquire TLS certificates, either from CA or create self-signed certificates.
```

Comment:
We can post a cross link to our own how to configure TLS for minio doc available already on docs.minio.io

Reply:
Done @deekoder